src/index.ts

```diff
@@ -228,6 +228,20 @@ function getConfigAttrNameCase() {
   return 'kebabCase' as const;
 }
 
+type DocumentDiagnosticsType = {
+  getDocumentVersionRequest: boolean;
+};
+
+function getConfigDiagnostics(): DocumentDiagnosticsType | undefined {
```

Comment:
Just a hint, you can use `NonNullable<shared.ServerInitializationOptions['languageFeatures']>['diagnostics']` instead of `DocumentDiagnosticsType` in return type.

Reply:
Thank!!